spec/lib/redmine/scm/adapters/github_adapter_spec.rb

```diff
@@ -3,51 +3,72 @@
 require File.expand_path('../../../../rails_helper', __dir__)
 
 RSpec.describe RedmineGithub::Scm::Adapters::GithubAdapter do
-  subject { RedmineGithub::Scm::Adapters::GithubAdapter.new(url, nil, login, token) }
-
+  let(:scm) { RedmineGithub::Scm::Adapters::GithubAdapter.new(url, nil, login, token) }
   let(:url) { 'https://github.com/company/repo.git' }
   let(:login) { 'user' }
   let(:token) { '1234567890' }
 
-  before do
+  before :each do
```

Comment:
I think this part should be moved in fetch_remote description

Reply:
Almost all command in the adapter are using `Dir.exist?()` and `git_cmd() `, so I put `allow()` in the before block